theano/sandbox/gpuarray/conv.py

```diff
@@ -127,11 +112,12 @@ def make_node(self, img, kern):
             raise TypeError('img must be 4D tensor')
         if kern.type.ndim != 4:
             raise TypeError('kern must be 4D tensor')
-        img = as_gpuarray_variable(img)
-        kern = as_gpuarray_variable(kern)
+        img = as_gpuarray_variable(img, self.context)
```

Comment:
Why this op need a context parameter? I do not see a need for it. We could use the default infer_context for this.